resources/skins.citizen.styles/components/Footer.less

```diff
@@ -134,6 +144,10 @@
 			background-color: var( --background-color-base-fixed ); // To prevent this being flipped in dark mode
 		}
 	}
+
+	&-tagline {
+		word-break: keep-all;
```

Comment:
To get the footer bottom wrapper work correctly for a long tagline, we can set a `flex-wrap: wrap` to the footer bottom so that it would format correctly